framework_tvb/tvb/interfaces/web/static/js/genericTVB.js

```diff
@@ -877,7 +877,7 @@ function waitForStorageUpdateToEnd() {
  *
  * @param projectId the selected project
  */
-function showDataUploadOverlay(projectId) {
+function showDataUploadOverlay(projectId, projectName) {
```

Comment:
This new argument doesn't seem to be used. I will remove it.

Reply:
I forgot it there...